src/utils/server-utils.ts

```diff
@@ -270,3 +278,119 @@ export async function getLayersAvailableDates(): Promise<AvailableDates> {
 
   return merge({}, ...layerDates);
 }
+
+/**
+ * Format value from featureInfo response based on LabelType provided
+ *
+ * @return a formatted string
+ */
+function formatFeatureInfo(value: string, type: LabelType): string {
+  if (type === LabelType.Date) {
+    return `${moment(value).utc().format('MMMM Do YYYY, h:mm:ss')} UTC`;
+  }
+
+  return value;
+}
+
+/**
+ * Executes a getFeatureInfo request
+ *
+ * @return object of key: string - value: string with formatted values given label type.
+ */
+async function runFeatureInfoRequest(
+  url: string,
+  wmsParams: requestFeatureInfo,
+  layers: WMSLayerProps[],
+): Promise<{ [name: string]: string }> {
+  // Transform to snake case.
+  const toSnakeWmsParams = Object.entries(wmsParams).reduce(
```

Comment:
Replace with lodash's snakeCase function.
https://lodash.com/docs/4.17.15#snakeCase

Reply:
included in 73987d1281c385364486401523595de3c0adf1d5

Reply:
Might be clearer if called `wmsParamsInSnakeCase`, what do you think?